### DISCUSSION TITLE
Navigation support AndroidTV

### DISCUSSION THREAD

**Discussion by txret** [KRTirtho/spotube#604] [Ideas]
As far as I can tell it is not currently  possible to navigate the UI on an AndroidTV device. I am unfamiliar with flutter development so I don't know what needs to be done to be able to navigate between the widgets with the D-pad nor to capture air mouse "pan" as scrolling (similar to "touch scrolling"). Are there any plans on adding this functionality or does anyone have experience and any pointers on what needs to be done? Would wrapping widgets with something like [dpad_container](https://github.com/nicoaudy/dpad_container) be a starting point?

Thanks for an otherwise great app!

**Comment by KRTirtho** [comment]
Unfortunately, both Flutter and Spotube doesn't support Android TV yet

**Reply by k8p8sz** [reply]
I saw a reddit discussion that with some changes on the code can help support the flutter based programs run normally on android TV. Have you considerred these? It would be awsome!
https://www.reddit.com/r/FlutterDev/comments/tcfgcq/lessons_learned_releasing_my_app_for_android_tv/
after installing it starts normally, only with the remote you cant move the focus, which issue is I think described in the above link. Thanks anyway for the incredible program!

**Reply by KRTirtho** [reply]
It's not some changes that are required. It's tons of changes.

**Reply by KRTirtho** [reply]
Changes at the framework level 😂

**Reply by k8p8sz** [reply]
it's a pity :), Thanks anyway for the reply!
